.env.example

```diff
@@ -69,11 +64,21 @@ ORIGINS=//localhost:*
 
 # You can configure the max run duration for jobs in milliseconds. This should
 # be lower than the pod termination grace period if using Kubernetes.
-WORKER_MAX_RUN_DURATION_SECONDS=60
-WORKER_MAX_RUN_MEMORY_MB=500
-WORKER_CAPACITY=4
+# WORKER_MAX_RUN_DURATION_SECONDS=60
 
-MAX_DATACLIP_SIZE_MB=10
+# TODO: these aren't specified in Runtime, do they belong to the worker process?
```

Comment:
yes! interesting point... we _SHARE_ config between worker and app. should we document that in lightning?

Reply:
Ah ok, so this is a bit of an unusual case - this _doesn't_ matter if you're deploying in a cluster/different images; and only really applies in self-contained environments (like dev and test).

So in this exact example, we're implicitly passing through env vars to the rtm process;
So we can make our own defaults inside the rtm process that pass down env vars to the rtm child process, which has a downside of overriding any default changes in the cli.
Or we leave it alone and expect if someone wants to change these settings to use env vars.
Or lastly we parse the env vars into lightnings config (under the `:workers` key) and conditionally set them in the child process env; which has little gain - but is more obvious and kinda 'self-documenting'.